tier1/checklist.md

```diff
@@ -0,0 +1,387 @@
+
+
+   #### Toolkit
+   Below is a list of suggested tools to run for code analysis:
+
+| **Tool** |  **Description** |  **Link**   |
+:-------| :-------------: | :------: |
+| Repo Linter  | Lint repositories for common issues such as missing files etc.| https://github.com/todogroup/repolinter |
+|  gitleaks  | Protect and discover secrets using Gitleaks :key:  |  https://github.com/gitleaks/gitleaks
+| git filter-repo |  Entirely remove unwanted files / files with sensitive data from a repository's history   |  https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository
+
+
+**Results**
+
+*Insert Review Here*
+
+----
+
+### Review Licensing
+
+Ensure that acceptable licensing is indicated for the project. Most often, software released as open source by the federal government is done so under the Creative Commons Zero 1.0 license.
+
+Suggested licensing:
+
+   **Public Domain**
+   
+  - This project is in the public domain within the United States, and copyright and related rights in the work worldwide are waived through the CC0 1.0 Universal public domain dedication.
+
+  - All contributions to this project will be released under the CC0 dedication. By submitting a pull request, you are agreeing to comply with this waiver of copyright interest.
+
+If your project is not being dedicated to the public domain under CC0, due to being work for hire, or some other documented reason, then choosing another [OSI approved license](https://opensource.org/license) is the next best thing.
+
+**Results**
+
+*Insert review here*
+
+---
+
+
+### Review Commit History 
+
+Review the history of commits to the version control system used, and whether the team prefers to clean (e.g.,rebase) this history before releasing to the public.
+
+If not rebasing, verify that:
+
+1. there are no obscene or impolite remarks in comments or commit history
+
+2. thee are no sensitive internal URLs/IP Addresses in comments or commit history 
+
+3. there are no credential files such as passwords, API/SSH/GPG keys checked into the repo.
+
+Consider using the following tools to perform the tasks above:
+
+[^1]: Table needs work - [x] 
+
+| Tool        | Description                                                      | Link                                    |
+|-------------|------------------------------------------------------------------|-----------------------------------------|
+| gitleaks | Open source tool that detects and prevents secrets (passwords/api/ssh keys) checked-in to your git repo | https://github.com/gitleaks/gitleaks   https://akachandwani.medium.com/what-is-gitleaks-and-how-to-use-it-a05f2fb5b034 |
+| git filter-repo    | Entirely remove unwanted files / files with sensitive data from a repository's history                     |    https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository                   |
+
+
+
```

Comment:
Please add a results section below the Review Commit history section:
Results
*Insert Review Here*